app/views/courses/search.html.erb

```diff
@@ -21,9 +18,25 @@ What you are looking for cannot be found.
         <td><%= Professor.find(course.professor_id).name %></td>
 
         <td><%= link_to 'Show', course %></td>
-        <td><%= link_to 'Edit', edit_course_path(course) %></td>
-        <td><%= link_to 'Destroy', course, method: :delete, data: { confirm: 'Are you sure?' } %></td>
       </tr>
     <% end %>
   </tbody>
 </table>
+
+<%= search_form_for @search do |f| %>
+  <fieldset>
+    <legend>User</legend>
+    <ul>
+      <li>
+        <%= f.label :name_cont %>
+        <%= f.text_field :name_cont %>
+      </li>
+      <li>
+        <%= f.label :crn_cont %>
+        <%= f.text_field :crn_cont %>
+      </li>
+    </ul>
+  </fieldset>
+
+  <%= f.submit "Search!" %>
+<% end %>
```

Comment:
we could probably stick this search code in a partial, given we're calling it a bunch from a bunch of different files. shared/_search_bar.html.erb maybe?